src/javascripts/templates/template.js

```diff
@@ -0,0 +1,70 @@
+import { formatNumber } from "../helpers/helpers"
+
+class Template {
+    /**
+     * Constructor of Template object
+     */
+    constructor(){ }
+
+     /**
+     * HTML Template for render an Transaction object
+     * @param {object} transaction 
+     */
+
+    static transaction = (transaction) => `
```

Comment:
function name should start with verb

Reply:
I fixed it @phuocduong-agilityio  . Please help me check. Thanks !